javagen/src/main/resources/Readme_protocol.txt

```diff
@@ -41,6 +41,21 @@ Various documentation is available to help you get started
 ```java {{package-name}}.readme
 ```
 
+### Service API versions
```

Comment:
wonder why do we put it under '##Example' section? Should it be '## Service API versions'?

Reply:
In discussion, Srikanta wanted this in example section
https://github.com/Azure/autorest.java/issues/2819#issuecomment-2176842642